dagshub/data_engine/model/query.py

```diff
@@ -14,10 +15,15 @@
 
 _metadataTypeLookup = {
     int: MetadataFieldType.INTEGER,
+    Int: MetadataFieldType.INTEGER,
     bool: MetadataFieldType.BOOLEAN,
+    Bool: MetadataFieldType.BOOLEAN,
     float: MetadataFieldType.FLOAT,
+    Float: MetadataFieldType.FLOAT,
     str: MetadataFieldType.STRING,
+    String: MetadataFieldType.STRING,
     bytes: MetadataFieldType.BLOB,
+    Blob: MetadataFieldType.BLOB,
 }
```

Comment:
I think this will break the reverse lookup table, because it will only have one of the types now (since the reverse is also a dict).

I think the "clean" way to solve this is to add a `classmethod` onto the dtypes that will return the MetadataFieldType, and when you want to check for the type do a `if isinstance(value, DagshubDataType)`

Hope it makes sense